mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -2581,7 +2581,8 @@ impl MarketCoinOps for TendermintCoin {
 
                 if let Some(tx_status_code) = tx_status_code {
                     return match tx_status_code {
-                        cosmrs::tendermint::abci::Code::Ok => Ok(()),
+                        // Tendermint uses Byzantine Fault Tolerance (BFT), achieve instant finality once a block is committed.
+                        cosmrs::tendermint::abci::Code::Ok => Ok(1u64),
```

Comment:
I think it's better this function `wait_for_confirmations` return Optional u8 or 16 and we can return None for `tendermint`. cc @onur-ozkan